android/app/src/main/java/com/created/team201/presentation/login/LoginActivity.kt

```diff
@@ -46,6 +53,16 @@ class LoginActivity : BindingActivity<ActivityLoginBinding>(R.layout.activity_lo
         }
     }
 
+    private fun observeGuestState() {
+        loginViewModel.signUpGuestState.observe(this) { guestState ->
+            when (guestState) {
+                SUCCESS -> navigateToMain()
+                FAIL -> {}
+                IDLE -> throw IllegalStateException()
```

Comment:
빈괄호 대신 `Unit`으로 쓰는게 조금 더 이쁘더라구요

또한 지금 현재 `guestState`가 explicit parameter로 존재하는데 언뜻 보면 게스트의 상태가 어떤 상태를 뜻하는 것인지 단번에 알아차리기가 힘들 것 같습니다. 

```suggestion
    private fun observeGuestState() {
        loginViewModel.signUpGuestState.observe(this) { signUpGuestState ->
            when (signUpGuestState) {
                SUCCESS -> navigateToMain()
                FAIL -> Unit
                IDLE -> throw IllegalStateException()
```

Reply:
+ 이쁘네요
+ 말씀들어보니 그런듯하네요 굳